Mlem/App Constants.swift

```diff
@@ -24,10 +24,16 @@ struct AppConstants {
 
     static let keychain: Keychain = .init(service: "com.hanners.Mlem-keychain")
 
-    // MARK: - DragGesture thresholds
-
-    static let longSwipeDragMin: CGFloat = 150
+    // MARK: - SwipeyView drag action thresholds
+    
+    static let swipeActionDragThresholds: [CGFloat] = [
```

Comment:
It does haha, AppConstants has so many things in it now anyway

Reply:
Moved values to `SwipeConfiguration.DragThresholds`:
- Added a `var userPreferred` that currently just returns Mlem default thresholds (so users can customize this, if we decide that's a thing)
- SwipeyView now calls `.userPreferred` instead of the hard-coded Mlem default thresholds.
- Added documentation.